checkov/gitlab/checks/two_factor_authentication.py

```diff
@@ -24,9 +24,9 @@ def __init__(self) -> None:
     def scan_entity_conf(self, conf: list[dict[str, Any]], entity_type: str) -> tuple[CheckResult, list[dict[str, Any]]] | None:  # type:ignore[override]
         if schema.validate(conf):
             for group in conf:
-                if group.get("require_two_factor_authentication", False) is True:
-                    return CheckResult.PASSED, conf
-            return CheckResult.FAILED, conf
+                if group.get("require_two_factor_authentication") is False:
```

Comment:
I think that maybe `require_two_factor_authentication` is not a required field (although the schema down here states it is required - from the docs: https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/api/groups.md) 
Hence in case there is no `require_two_factor_authentication` the schema will be filtered and will not get here

Reply:
Looks like the docs state that `require_two_factor_authentication` is not mandatory on PUT/POST operations, since it defaults to False. But on retrieval it is always returned.

Reply:
You are correct! missed the difference :)